packages/kbn-palettes/classes/types.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export type KbnPaletteType = 'categorical' | 'gradient';
+
+/**
+ * Common palette definition used throughout kibana
+ */
+export interface IKbnPalette {
+  /**
+   * Unique identifier for the palette
+   */
+  id: string;
+  /**
+   * Display name of this palette.
+   */
+  name: string;
+  /**
+   * Type of pallette
+   */
+  type: KbnPaletteType;
+  /**
+   * Number of colors to display
+   */
+  colorCount: number;
+  /**
+   * Palette belongs to an outdated theme set
+   */
+  legacy: boolean;
+  /**
+   * Alternate aliases/ids this palette matches
+   */
+  aliases: string[];
+  /**
+   * Excluded from `getAll` but can still query for palette with `get`/`query`
```

Comment:
ca you

Reply:
Yup 👍🏼  016f1615074a66354d91174004f5b9ea31ef4a56